addons/google_calendar/models/google_calendar.py

```diff
@@ -1,22 +1,20 @@
 # -*- coding: utf-8 -*-
 
+from datetime import datetime, timedelta
+import logging
 import operator
 import simplejson
 import urllib2
 
+from dateutil import parser
+import pytz
+
 import openerp
```

Comment:
I see "openerp" is imported only to access "openerp.modules.registryRegistryManager", so why not import it only, like,
from openerp.modules.registry import RegistryManager